MixItUp.Base/Actions/OvrStreamAction.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Mixer.Base.Util;
+using MixItUp.Base.ViewModel.User;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MixItUp.Base.Actions
+{
+    public enum OvrStreamActionTypeEnum
+    {
+        [Name("Show")]
```

Comment:
Maybe include the full text of the action for these? (EX: "Show" => "Show Title")